yarn-project/pxe/src/contract_function_simulator/oracle/oracle.ts

```diff
@@ -26,60 +26,80 @@ export class Oracle {
   }
 
   toACIRCallback(): ACIRCallback {
```

Comment:
now this function also checks that the oracle functions here are prefixed either with `pxe` or `utility`.

+ polished it a bit